src/types/onyx/SearchResults.ts

```diff
@@ -180,7 +180,7 @@ type SearchTransaction = {
     tag: string;
 
     /** The transaction description */
-    comment: {
+    comment?: {
         /** Content of the transaction description */
         comment: string;
```

Comment:
Isn't this one also optional? Or do we not need to mark it optional since the parent property is optional?

Reply:
I think we should mark nested properties as optional for clarity and to ensure type safety, even if the parent property is already optional. However, open to other suggestions :D

Reply:
I agree, that sounds the most clear 👍

Reply:
Okay, I will tweak it 🙂

Reply:
Done - 2b8780c.